sgt-back-end/index.js

```diff
@@ -0,0 +1,133 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const jsonMiddleWare = express.json();
+
+app.use(jsonMiddleWare);
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    select *
+      from grades
+  `;
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      res.status(200).json(grades);
+    })
+    .catch(error => {
+      // eslint-disable-next-line no-console
+      console.log(error);
+      res.status(500).json({ error: 'An unexpected error has occurred' });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const body = req.body;
+  const score = Number(req.body.score);
+  if (!body.name || !body.course || !body.score) {
+    res.status(400).json({ error: 'Entry must contain a "name", "course", and "score"' });
+    return;
+  } else if (!score || score > 100) {
```

Comment:
Here you also want to verify `score` is a whole number (integer), you can use `Number.isInteger(score)`